src/DistributedComputations/distributed_fft_based_poisson_solver.jl

```diff
@@ -200,8 +200,8 @@ function validate_global_grid(global_grid::RectilinearGrid)
                Please rotate the domain to obtain the required topology")
     end
     
-    if !(global_grid isa XYZRegularRG) 
-        throw("Stretched directions are not supported with distributed grids at the moment.")
+    if !(global_grid isa YZRegularRG) && !(global_grid isa XYRegularRG) && !(global_grid isa XZRegularRG) 
+        throw(ArgumentError("Only stretching in one direction is supported with distributed grids at the moment."))
```

Comment:
This error doesn't belong here. 

The error message should say something like

> X type does not support `RectilinearGrid` on `Distributed` architectures that have variably-spaced cells in more than one direction. Cells may be variably-spaced in at most one direction.

In other words you have to explain 1) where the problem arises specifically (`ArgumentError` should arise from a constructor ideally to drive this home) and 2) specifically what the issue is. "stretching in one direction" is too vague.

Reply:
Should be resolved